### PR TITLE
add select_genes_by_gamma_r2 in cell_velocities

### PR DESCRIPTION
**Pull request by dummyindex**

### CODE REVIEW

File: dynamo/plot/utils.py [hunk 1]
```diff
@@ -580,7 +580,7 @@ def _matplotlib_points(
             ax.legend(
                 handles=legend_elements,
                 bbox_to_anchor=(1.04, 1),
-                loc=show_legend,
+                # loc=show_legend,
```
Comment (Xiaojieqiu) [inline dynamo/plot/utils.py:583]: why do you comment this?

File: dynamo/tools/utils.py [hunk 2]
```diff
@@ -181,6 +184,39 @@ def index_gene(adata, arr, genes):
             return arr[:, mask]
 
 
+def select_genes_by_gamma_r2(
```
Comment (Xiaojieqiu) [inline dynamo/tools/utils.py:187]: doc

File: dynamo/configuration.py [hunk 3]
```diff
@@ -49,12 +49,28 @@ class DynamoAdataConfig:
 
     config_key_to_values = None
 
-    def check_config_var(val, key):
+    def check_config_var(val, key, replace_val=None):
```
Comment (Xiaojieqiu) [inline dynamo/configuration.py:52]: doc

File: dynamo/tools/utils.py (hunk 2 above)
Comment (dummyindex) [reply dynamo/tools/utils.py:187]: Already exists below

File: dynamo/plot/utils.py (hunk 1 above)
Comment (dummyindex) [reply dynamo/plot/utils.py:583]: fixed by Xiaojie in the separate commits.

File: dynamo/tools/utils.py [hunk 4]
```diff
@@ -181,6 +184,39 @@ def index_gene(adata, arr, genes):
...
+    adata: AnnData, var_store_key: str, minimal_gene_num: Union[int, None] = 50
+) -> Union[list, pd.Series, np.array]:
+    """When the sum of `adata.var[var_store_key]` is less than `minimal_gene_num`, select the `minimal_gene_num` genes and save to (update) adata.var[var_store_key].
+
+    Parameters
+    ----------
+        adata:
```
Comment (Xiaojieqiu) [inline dynamo/tools/utils.py:194]: those should be also documented

**Pull request merged**